src/store/customer/actions.js

```diff
@@ -51,3 +51,52 @@ export function regCustomer({commit}, payload) {
       commit('setIsLoading', false)
     })
 }
+
+export function loginCustumerFB({commit}, payload) {
```

Comment:
why use loginCustumerFB, can just use loginFb(), which call to the mutation loginFb